src/Workspaces/Core/Portable/LanguageServices/SyntaxFactsService/ISyntaxFactsServiceExtensions.cs

```diff
@@ -139,5 +139,16 @@ public static bool IsAnonymousOrLocalFunctionStatement(this ISyntaxFactsService
             => syntaxFacts.IsAnonymousFunction(node) ||
                syntaxFacts.IsLocalFunctionStatement(node);
 
+        public static SyntaxNode GetExpressionOfElementAccessExpression(this ISyntaxFactsService syntaxFacts, SyntaxNode node)
+        {
+            syntaxFacts.GetPartsOfElementAccessExpression(node, out var expression, out _);
+            return expression;
+        }
+
+        public static SyntaxNode GetArgumentListOfElementAccessExpression(this ISyntaxFactsService syntaxFacts, SyntaxNode node)
```

Comment:
>GetArgumentListOfElementAccessExpression [](start = 33, length = 40)

These two methods appear unused.

Reply:
Both methods are used.  My change in this PR is to make these extension that then operate off a new SyntaxFact (which itself is used directly in this PR).